blocks/library/freeform/old-editor.js

```diff
@@ -67,6 +70,16 @@ export default class OldEditor extends Component {
 			} );
 		} );
 
+		editor.on( 'keydown', ( event ) => {
+			if ( ( event.keyCode === BACKSPACE || event.keyCode === DELETE ) &&
+					/^\n?$/.test( editor.getContent( { format: 'text' } ) ) ) {
```

Comment:
How performant is `{ format: 'text' }`? Does it need to run TinyMCE's serializer? If so, might be worryingly non-performant to occur on every keypress.